hw3/src/main/resources/templates/city_details.html

```diff
@@ -42,5 +42,19 @@ <h2>City</h2>
             </tr>
         </tbody>
     </table>
+
+    <p>For a fabulous weekend get away for 2 to [[${cityInfo.name}]] sign up here.</p>
+    <form th:action="@{/cities/reservation}" method="post">
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email" />
+        <br />
+
+        <div><input type="radio" name="level" value="luxury" />Luxury $1000</div>
+        <div><input type="radio" name="level" value="very good" />Very Good $500</div>
+        <div><input type="radio" name="level" value="good" checked="checked" />Good $350</div>
+
+        <input style="display: none" type="text" name="city" th:value="${cityInfo.name}" />
```

Comment:
I did something similar but made it hidden instead.  Good to know that there are other alternative options.